doc/tutorials/rt_linux.rst

```diff
@@ -44,71 +44,135 @@ system on Intel KBL NUC with a SATA SSD as ``/dev/sda`` and an NVME SSD as
+
+   #. Modify ``launch_hard_rt_vm.sh`` script::
+         
+         # vim /usr/share/acrn/samples/nuc/launch_hard_rt_vm.sh        
+      
+         <Modify the passthru_bdf and passthru_vpid with your NVMe pass-through IDs>
+         
+         passthru_vpid=(
+         ["eth"]="8086 156f"
+         ["sata"]="8086 9d03"
+         )
+         passthru_bdf=(
+         ["eth"]="0000:00:1f.6"
+         ["sata"]="0000:00:17.0"
+         )
+
+         TO:
+         passthru_vpid=(
+         ["eth"]="8086 156f"
+         ["sata"]="8086 f1a6"
+         )
+         passthru_bdf=(
+         ["eth"]="0000:00:1f.6"
+         ["sata"]="0000:01:00.0"
+         )
+
+         <Modify NVMe pass-through id>
+         
+         -s 2,passthru,0/17/0 \
+      
+         TO:
+         -s 2,passthru,01/00/0 \
+      
+         <Modify rootfs to NVMe>
+         
+         -B "root=/dev/sda3 rw rootwait maxcpus=$1 nohpet console=hvc0 \
+      
+         TO:
+         -B "root=/dev/nvme0n1p3 rw rootwait maxcpus=$1 nohpet console=hvc0 \ 
+
+#. Get IP address in real-time VM if you need it (There is no IP by default)
+
+   #. Method 1 ``virtio-net NIC``::
+
+         # vim /usr/share/acrn/samples/nuc/launch_hard_rt_vm.sh
+         
+         <add below line into acrn-dm boot args>
+         
+         -s 4,virtio-net,tap0 \
+
+   #. Method 2 ``pass-through NIC``::
+         
+         <Get your ethernet IDs first(in our example they are ``[00:1f.6]`` and ``[8086:15e3]``)>
+         
+         # lspci -nn | grep Eth
+         00:1f.6 Ethernet controller [0200]: Intel Corporation Ethernet Connection (5) I219-LM [8086:15e3]
+
+         # vim /usr/share/acrn/samples/nuc/launch_hard_rt_vm.sh
+
+         <Modify the passthru_bdf and passthru_vpid with your ethernet IDs>
+         
+         passthru_vpid=(
+         ["eth"]="8086 156f"
+         ["sata"]="8086 f1a6"
+         )
+         passthru_bdf=(
+         ["eth"]="0000:00:1f.6"
+         ["sata"]="0000:01:00.0"
+         )
+
+         TO:
+         passthru_vpid=(
+         ["eth"]="8086 15e3"
+         ["sata"]="8086 f1a6"
+         )
+         passthru_bdf=(
+         ["eth"]="0000:00:1f.6"
+         ["sata"]="0000:01:00.0"
+         )
+
+         <Remove the # in the following three lines>
+         
+         #echo ${passthru_vpid["eth"]} > /sys/bus/pci/drivers/pci-stub/new_id
+         #echo ${passthru_bdf["eth"]} > /sys/bus/pci/devices/${passthru_bdf["eth"]}/driver/unbind
+         #echo ${passthru_bdf["eth"]} > /sys/bus/pci/drivers/pci-stub/bind
+
+         TO:
+         echo ${passthru_vpid["eth"]} > /sys/bus/pci/drivers/pci-stub/new_id
+         echo ${passthru_bdf["eth"]} > /sys/bus/pci/devices/${passthru_bdf["eth"]}/driver/unbind
+         echo ${passthru_bdf["eth"]} > /sys/bus/pci/drivers/pci-stub/bind
+
+         <add below line into acrn-dm boot args,behind is your ethernet ID>
+         
+         -s 4,passthru,00/1f/6 \      
+
+   .. note::
+      
+      method 1 can both get sos and uos ip.
```

Comment:
I prefer to talk about network connectivity instead of IP address. How about the following lines:
```
Method 1 will give both the Service VM and User VM network connectivity
Method 2 will give the User VM a network interface, the Service VM will loose it
```
@dbkinder may have some even better wording to suggest here.